src/routes/index.js

```diff
@@ -41,5 +42,6 @@ router.use('/users', validateToken, userRouter);
 router.use('/channels', validateToken, channelRouter);
 router.use('/clients', validateToken, clientRouter);
 router.use('/campaigns', validateToken, campaigntRouter);
+router.use('/notifications', validateToken, notificationRouter);
```

Comment:
@Alexjes98 if notification are sent by recipient that route might be like this:

`/recipeints/:idRecipients/notifications` OR `/notifications?recipient=recipientId` OR 
`/users/:userId/notifications`